### DISCUSSION TITLE
Mod that changes only the visual size of hit objects

### DISCUSSION THREAD

**Discussion by LittleEndu** [ppy/osu#14369] [Ideas]
I have always thought one's aim would improve if the object you are trying to aim seems smaller than its hitbox is. You might be able to achieve this (later) with skinning, but changing only the visual size of hit objects could also work as a mod. 

As for the implementation, currently there seems to be some objects that have separate drawable components for detecting their hits, like `DrawableHitCircle`'s `HitReceptor`, but there are many others, like `SliderBall` that seem to use their own drawable for detecting hits. Also for some drawables it becomes a question of what value is safe to change, like you would want to change the `PathRadius` for sliders but that change gets undone when the scale of the slider changes.

**Comment by peppy** [comment]
I'd say this is best left for skinning, personally.
